Joint deliverables

anssiko opened the pull request:
Related #123


<!--
    This comment and the below content is programmatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://pr-preview.s3.amazonaws.com/w3c/das-charter/pull/126.html" title="Last updated on Jun 20, 2023, 4:25 PM UTC (636ce34)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/das-charter/126/05be7da...636ce34.html" title="Last updated on Jun 20, 2023, 4:25 PM UTC (636ce34)">Diff</a>

anssiko committed:
Add Contact Picker API as a joint deliverable

anssiko commented:
This PR is based on the [current DAS WG charter](https://www.w3.org/2022/11/das-wg-charter.html) and is synchronized with the [current Web Apps WG charter](https://www.w3.org/2022/04/webapps-wg-charter.html) and thus adds Contact Picker API as a joint deliverable.

Other joint deliverables may be added subject to bilateral discussions.

See also: [HTML diff](https://services.w3.org/htmldiff?doc1=https%3A%2F%2Fwww.w3.org%2F2022%2F11%2Fdas-wg-charter.html&doc2=https%3A%2F%2Fdeploy-preview-126--w3c-das-charter.netlify.app%2Fdas-wg-charter) between the current DAS WG charter and this PR.

anssiko committed:
Add Screen Wake Lock API as a joint deliverable

Also make the joint deliverable status Notes more prominent.

tomayac reviewed (approved):
Non-owner LGTM

marcoscaceres commented on das-wg-charter.html:
```diff
@@ -574,6 +577,9 @@ <h3 id="tentative">
               <b>Adopted Draft</b>: <a href=
               "https://wicg.github.io/contact-api/spec/">Adopted from WICG</a>
             </p>
+            <p class="issue">
+              <b>Note:</b> This work is a possible joint deliverable with the <a href="https://www.w3.org/groups/wg/webapps">Web Applications Working Group</a>.
+            </p>
```
This is already a fact: 

```suggestion
            <p>
              This work is a joint deliverable with the <a href="https://www.w3.org/groups/wg/webapps">Web Applications Working Group</a>.
            </p>
```

See below from what's on TR:

<img width="863" alt="Screenshot 2023-04-20 at 11 24 49 am" src="https://user-images.githubusercontent.com/870154/233233860-898d7e7a-80d9-42a6-b97c-7fd571c4a4e5.png">

anssiko replied:
Fixed by 3dc341b

anssiko committed:
Make Contact Picker API an actual joint deliverable

Remove "possible", entered TR on 2022-12-20

marcoscaceres commented on das-wg-charter.html:
```diff
@@ -259,7 +259,10 @@ <h3 id="rec-track">
             </p>
-          </dd>
+            <p class="issue">
+              <b>Note:</b> This work is a possible joint deliverable with the <a href="https://www.w3.org/groups/wg/webapps">Web Applications Working Group</a>.
```
```suggestion
              <b>Note:</b> This work is a joint deliverable with the <a href="https://www.w3.org/groups/wg/webapps">Web Applications Working Group</a>.
```

anssiko replied:
Fixed by https://github.com/w3c/das-charter/pull/126/commits/14e2ce982f668c345227154ce06368f6445e8a47

marcoscaceres commented:
could we make it so that this PR only includes the joint deliverable? It will make it easier to review on our side 🙏

reillyeon commented:
> could we make it so that this PR only includes the joint deliverable? It will make it easier to review on our side 🙏

+1, moving around Compute Pressure makes the diff harder to read given the HTML preview seems to be broken.

anssiko committed:
Make Screen Wake Lock API an actual joint deliverable

Remove "possible", entered TR on 2015-02-12

anssiko commented:
Moved Compute Pressure changes to PR #128, updated this PR accordingly and addressed comments.

@himorin can you please fix the HTML preview and diff.

reillyeon reviewed (approved):
Thanks for the updates. This looks good to me.

himorin commented:
> @himorin can you please fix the HTML preview and diff.

@anssiko .pr-preview.json itself does not exist... I'll file a PR to setup pr-preview, but was it really worked for this repository!!??

marcoscaceres reviewed (changes requested):
This should list: 

 * Geolocation
 * Screen Wake Lock
 * DeviceOrientation Event 

As joint deliverables.

marcoscaceres commented:
I think we are still waiting on the charter update to list the above joint deliverables with WebApps WG.

marcoscaceres commented:
Just FYI, here is the proposed Web Apps Charter listing the join deliverables: 
https://w3c.github.io/webappswg/charter/draft-charter-2023.html 

It's expected that the updated charter will go for review to the AC in June, so would be great to have this one up to date also.

reillyeon committed:
Add Geolocation and DeviceOrientation Events as joint deliverables

Matches the joint deliverables from the proposed Web Applications Working Group charter: https://w3c.github.io/webappswg/charter/draft-charter-2023.html

reillyeon commented:
@anssiko, given I just modified this change I will wait for your approval as co-chair before merging.

anssiko commented:
@reillyeon, https://github.com/w3c/das-charter/pull/126/commits/bd57663b4e584e1a3475bad81ee935f64d57ff18 and https://w3c.github.io/webappswg/charter/draft-charter-2023.html refer to a different Geolocation spec.

I believe WebApps WG's aspiration is to help with the maintenance of the Geolocation API that has reached the W3C Recommendation status (and is revised per https://www.w3.org/2023/Process-20230612/#revising-rec). Recommendations are listed under the Maintenance section of the charter.

reillyeon committed:
Oops, wrong Geolocation API

reillyeon commented:
> @reillyeon, [bd57663](https://github.com/w3c/das-charter/commit/bd57663b4e584e1a3475bad81ee935f64d57ff18) and https://w3c.github.io/webappswg/charter/draft-charter-2023.html refer to a different Geolocation spec.

Oops, fixed.
